release/RELEASE.md

```diff
@@ -121,24 +55,20 @@ patches we backport to the 1.0 release branch).
    where `$VERSION` is the full version tag of the new release you are making,
    i.e. the second column of the `LATEST` file.
 
-1. Windows prerequisites for running the tests:
+1. Prerequisites for running the tests:
     - [Visual Studio Code, Java-SDK](https://docs.daml.com/getting-started/installation.html)
-      - The above link takes you docs.daml.com's "getting started" installation guide;
-    - [Maven](https://maven.apache.org/install.html)
-      - You may have to manually set the  environment variable `JAVA_HOME`;
-      - For example, assuming the Zulu Java-SDK, something like
-        `C:\Program Files\Zulu\zulu-14`;
     - [Node.js](https://nodejs.org/en/download/)
-      - Just the bare install; you don't need Visual Studio build
-        tools for compiling C dependencies (and trying to install them
-        takes forever and in the end hangs it seems);
+      - Just the bare install; no need to build C dependencies.
```

Comment:
Does the yarn install not bundle nodejs on Windows?

Reply:
I followed the instructions here so not sure. However, the node install is ~60MB and the yarn install is ~1.5MB, so my first guess would be no.